pyanaconda/modules/storage/devicetree/viewer.py

```diff
@@ -189,7 +189,8 @@ def _get_format_data(self, fmt):
         data.mountable = fmt.mountable
         data.description = fmt.name or ""
 
-        # Collect the additional attributes.
+        # Collect the a_ditional attributes.
+        data.attrs["has_key"] = self._get_attribute(fmt, "has_key")
```

Comment:
If you want to test this, maybe:
- In `test_get_format_data` add the right parameter to `fmt2` construction.
- This needs further work in the `get_format` blivet function in `blivet/formats/__init__.py` to know what to pass so that the key is mock-present
- Then add the `has_key` thing to `assert self.interface.GetFormatData("dev2") == {`

I didn't find anything better yet...